three-cards-divination/src/main/resources/data.sql

```diff
@@ -0,0 +1,83 @@
+    ('Queen of Wands', 13, 'Minor Arcana', 'w13.jpg', '{"This card represents a woman with an attractive, appealing personality, likely born between March 11th and April 20th, who wants to charm you into doing things her way"}'),
+    ('King of Wands', 14, 'Minor Arcana', 'w14.jpg', '{"This card represents an older man with a commanding, charismatic personality, likely born between November 13th and December 12th, who prefers to give directions and have them followed"}'),
+    ('Ace of Pentacles', 1, 'Minor Arcana', 'p01.jpg', '{"Your health will improve", "The check you''re looking for really is in the mail"}'),
+    ('Two of Pentacles', 2, 'Minor Arcana', 'p02.jpg', '{"It''s time to balance the budget", "Avoid the temptation to spend critical funds on frivolous goods"}'),
+    ('Three of Pentacles', 3, 'Minor Arcana', 'p03.jpg', '{"A high-dollar contract is in your future", "If you work hard, you''ll succeed"}'),
+    ('Four of Pentacles', 4, 'Minor Arcana', 'p04.jpg', '{"A rainy day is coming—it''s time to save"}'),
+    ('Five of Pentacles', 5, 'Minor Arcana', 'p05.jpg', '{"Finances are getting tighter", "Prepare for a setback"}'),
+    ('Six of Pentacles', 6, 'Minor Arcana', 'p06.jpg', '{"When you need help, ask for it", "Remember, though: what you receive may be limited by what you''ve given to others in the past"}'),
+    ('Seven of Pentacles', 7, 'Minor Arcana', 'p07.jpg', '{"Things won''t work out as expected", "Pick up the pieces and prepare to move on"}'),
+    ('Eight of Pentacles', 8, 'Minor Arcana', 'p08.jpg', '{"Stop over-analyzing, researching, and outlining", "Buckle down and get the work done"}'),
+    ('Nine of Pentacles', 9, 'Minor Arcana', 'p09.jpg', '{"Until you appreciate what you have, you won''t have any luck getting more"}'),
+    ('Ten of Pentacles', 10, 'Minor Arcana', 'p10.jpg', '{"Big money is in the near future", "Expect a powerful blessing to come your way"}'),
+    ('Page of Pentacles', 11, 'Minor Arcana', 'p11.jpg', '{"This card represents a young man or woman with an earthy, practical demeanor, likely born an Aries, Taurus, or Gemini, who playfully encourages you to take financial or sexual risks"}'),
+    ('Knight of Pentacles', 12, 'Minor Arcana', 'p12.jpg', '{"A stingy person may chide you for spending money", "Be prepared to defend an economic or sexual decision"}'),
+    ('Queen of Pentacles', 13, 'Minor Arcana', 'p13.jpg', '{"This card represents a woman with an expansive, sensual nature, likely born between December 13th and 31st, who uses sensual appeal and the promise of reward to sway others to her point of view"}'),
+    ('King of Pentacles', 14, 'Minor Arcana', 'p14.jpg', '{"This card represents an older man with a financially, socially, and politically conservative spirit, likely born between August 12th and September 11th, who is known for putting his money where his mouth is"}')
```

Comment:
**Suggestion:** Use consistent quotation marks for string values [best practice]
```suggestion
    5 +    ("The Fool", 0, "Major Arcana", "m00.jpg", '["Watch for new projects and new beginnings", "Prepare to take something on faith", "Something new comes your way; go for it"]'),
    6 +    ("The Magician", 1, "Major Arcana", "m01.jpg", '["A powerful man may play a role in your day", "Your current situation must be seen as one element of a much larger plan"]'),
    7 +    ("The High Priestess", 2, "Major Arcana", "m02.jpg", '["A mysterious woman arrives", "A sexual secret may surface", "Someone knows more than he or she will reveal"]'),
    8 +    ("The Empress", 3, "Major Arcana", "m03.jpg", '["Pregnancy is in the cards", "An opportunity to be involved in luxurious sexuality is coming", "Beware a tendency toward addiction"]'),
    9 +    ("The Emperor", 4, "Major Arcana", "m04.jpg", '["A father figure arrives", "A new employer or authority figure will give you orders", "Expect discipline or correction in the near future"]'),
    10 +    ("The Hierophant", 5, "Major Arcana", "m05.jpg", '["Expect to be caught in a misdeed and punished accordingly", "Pray for forgiveness and confess wrongdoings", "A more experienced man, spiritual leader, or father figure will come into your life"]'),
    11 +    ("The Lovers", 6, "Major Arcana", "m06.jpg", '["A new personal or professional relationship blossoms", "Sexual opportunities abound", "Unexpectedly, a friend becomes a lover"]'),
    12 +    ("The Chariot", 7, "Major Arcana", "m07.jpg", '["Victory is a certainty", "Move ahead with all plans", "Beware the jealousy of others"]'),
    13 +    ("Strength", 8, "Major Arcana", "m08.jpg", '["Your self-control will be tested", "A woman will seek to change her partner or lover", "You are a strong, capable person"]'),
    14 +    ("The Hermit", 9, "Major Arcana", "m09.jpg", '["A period of loneliness begins", "One partner in a relationship departs", "A search for love or money proves fruitless"]'),
    15 +    ("Wheel of Fortune", 10, "Major Arcana", "m10.jpg", '["Some events are in the hands of heaven", "You\'ve lived through this before", "What happened then?"]'),
    16 +    ("Justice", 11, "Major Arcana", "m11.jpg", '["A legal verdict will be rendered soon", "Someone is making a decision", "You need to get the facts"]'),
    17 +    ("The Hanged Man", 12, "Major Arcana", "m12.jpg", '["A traitor is revealed", "One of your friends is working against you", "Change your ways or suffer the consequences"]'),
    18 +    ("Death", 13, "Major Arcana", "m13.jpg", '["A relationship or illness ends suddenly", "Limit travel and risk-taking", "General gloom and doom"]'),
    19 +    ("temperance", 14, "Major Arcana", "m14.jpg", '["Someone\'s using drugs or alcohol to excess", "It\'s time to get back on that diet"]'),
    20 +    ("The Devil", 15, "Major Arcana", "m15.jpg", '["Adultery and unfaithfulness", "A string of extremely bad luck is coming your way", "Beware evil influences and wolves in sheep\'s clothing"]'),
    21 +    ("The Tower", 16, "Major Arcana", "m16.jpg", '["Impending disaster", "Cancel plans and reverse decisions", "Someone wants to take you down a notch or two", "Don\'t hold back; say what you really mean"]'),
    22 +    ("The Star", 17, "Major Arcana", "m17.jpg", '["Get an astrology chart drawn up", "Someone is a little too starstruck", "What\'s happening now has long been fore-ordained"]'),
    23 +    ("The Moon", 18, "Major Arcana", "m18.jpg", '["Watch for problems at the end of the month", "Someone you know needs to howl at the moon more often", "Someone is about to change his or her mind about an important decision"]'),
    24 +    ("The Sun", 19, "Major Arcana", "m19.jpg", '["Everything\'s coming up roses (or sunflowers, whatever the case may be)", "Whatever\'s on your mind, go for it because you can\'t lose today"]'),
    25 +    ("Judgement", 20, "Major Arcana", "m20.jpg", '["An old issue you thought was over will come up again today", "Get ready for huge changes: break-ups, sudden calls from old friends, and unexpected setbacks", "God\'s trying to get your attention"]'),
    26 +    ("The World", 21, "Major Arcana", "m21.jpg", '["Winning the lottery", "Getting your heart\'s desire", "Having everything you ever imagined having"]'),
    27 +    ("Ace of Cups", 1, "Minor Arcana", "c01.jpg", '["Romance is in the cards", "A new relationship or marriage is just around the corner", "Prayers are answered"]'),
    28 +    ("Two of Cups", 2, "Minor Arcana", "c02.jpg", '["Someone has a secret crush on you", "Relationships should be mutual; get rid of a leech"]'),
    29 +    ("Three of Cups", 3, "Minor Arcana", "c03.jpg", '["Unconventional romance is coming your way: a love affair with someone you\'ve always dismissed"]'),
    30 +    ("Four of Cups", 4, "Minor Arcana", "c04.jpg", '["A lover is getting restless", "Find out what he or she needs, or new opportunities may lure your partner away"]'),
    31 +    ("Five of Cups", 5, "Minor Arcana", "c05.jpg", '["A breakup looms", "Don\'t cry over spilt milk", "Take your lumps and get back in the saddle"]'),
    32 +    ("Six of Cups", 6, "Minor Arcana", "c06.jpg", '["A stingy spirit is strangling your enjoyment of life", "Loosen up and think of others for once, why don\'t you?"]'),
    33 +    ("Seven of Cups", 7, "Minor Arcana", "c07.jpg", '["You\'re being fed a line", "Rather than be dazzled by fancy words and promises, demand something real"]'),
    34 +    ("Eight of Cups", 8, "Minor Arcana", "c08.jpg", '["Someone\'s \'\'stepping out\'\' on you, now or in the near future", "Maybe it\'s time to quit talking about the problem and just move on"]'),
    35 +    ("Nine of Cups", 9, "Minor Arcana", "c09.jpg", '["Whatever you want, you\'ll get it"]'),
    36 +    ("Ten of Cups", 10, "Minor Arcana", "c10.jpg", '["Marriage and family are in the cards", "Expect a friendship to blossom into a romance"]'),
    37 +    ("Page of Cups", 11, "Minor Arcana", "c11.jpg", '["This card represents a young man or woman with a watery, dreamy demeanor, likely born a Libra, Scorpio, or Sagittarius, who wants to start a new relationship with you"]'),
    38 +    ("Knight of Cups", 12, "Minor Arcana", "c12.jpg", '["This card represents a man with an emotional, sensitive personality, likely born between October 13th and November 11th, who wants you to rally around his latest passionate cause"]'),
    39 +    ("Queen of Cups", 13, "Minor Arcana", "c13.jpg", '["This card represents a woman with an emotional, deeply spiritual nature, likely born between June 11th and July 11th, who uses emotional and spiritual appeals to sway others to her point of view"]'),
    40 +    ("King of Cups", 14, "Minor Arcana", "c14.jpg", '["This card represents an older man with a gentle, sensitive presence, likely born between February 9th and March 10th, who is known for his fairness and tolerance"]'),
    41 +    ("Ace of Swords", 1, "Minor Arcana", "s01.jpg", '["The time to make a choice is now", "Stop wavering and do what you know is best"]'),
    42 +    ("Two of Swords", 2, "Minor Arcana", "s02.jpg", '["Sometimes, the only way to win is to refuse to fight", "You\'re stuck for now; let time pass before taking action"]'),
    43 +    ("Three of Swords", 3, "Minor Arcana", "s03.jpg", '["Breakups and infidelity abound", "What hurts now, though, will turn out to be good for you later on"]'),
    44 +    ("Four of Swords", 4, "Minor Arcana", "s04.jpg", '["Don\'t make any decision now", "Wait, and you\'ll be glad you did"]'),
    45 +    ("Five of Swords", 5, "Minor Arcana", "s05.jpg", '["Someone is stealing from you, financially or romantically", "Be wary of friends who talk behind your back"]'),
    46 +    ("Six of Swords", 6, "Minor Arcana", "s06.jpg", '["You\'ll soon go on a long journey over water", "Actions have unexpected consequences, so be prepared"]'),
    47 +    ("Seven of Swords", 7, "Minor Arcana", "s07.jpg", '["Don\'t assume people around you are worthy of your trust", "Ask for an accounting of where people have been, and what they\'ve been doing"]'),
    48 +    ("Eight of Swords", 8, "Minor Arcana", "s08.jpg", '["Get over playing the victim", "Once you realize you are your own biggest obstacle, nothing can hold you back"]'),
    49 +    ("Nine of Swords", 9, "Minor Arcana", "s09.jpg", '["If you take the action you\'re considering now, you\'ll be sorry in the future"]'),
    50 +    ("Ten of Swords", 10, "Minor Arcana", "s10.jpg", '["Disaster", "Put off plans and do not take action until omens are better"]'),
    51 +    ("Page of Swords", 11, "Minor Arcana", "s11.jpg", '["This card represents a young man or woman with an airy, intellectual demeanor, likely born a Capricorn, Aquarius, or Pisces, who wants to learn something new from you or have a discussion with you"]'),
    52 +    ("Knight of Swords", 12, "Minor Arcana", "s12.jpg", '["A blunder leads someone to say something he or she regrets", "If this was you, be prepared to apologize and move on"]'),
    53 +    ("Queen of Swords", 13, "Minor Arcana", "s13.jpg", '["This card represents a woman with an artistic, intellectual nature, likely born between September 12th and October 12th, who uses clever, positive communication to sway others to her point of view"]'),
    54 +    ("King of Swords", 14, "Minor Arcana", "s14.jpg", '["This card represents an older man with an insightful, deliberate spirit, likely born between May 11th and June 10th, who is known for his integrity and sharp decision-making ability"]'),
    55 +    ("Ace of Wands", 1, "Minor Arcana", "w01.jpg", '["Someone has the \'\'hots\'\' for you", "A new job offer is coming your way", "Walk softly, and carry a big stick"]'),
    56 +    ("Two of Wands", 2, "Minor Arcana", "w02.jpg", '["Beware false friends", "Don\'t be mealy-mouthed; say what you think and do what you want to do"]'),
    57 +    ("Three of Wands", 3, "Minor Arcana", "w03.jpg", '["You\'ll be planning a trip soon", "Be on the lookout: your ship is coming in"]'),
    58 +    ("Four of Wands", 4, "Minor Arcana", "w04.jpg", '["Someone is watching and evaluating your work", "You may get a wedding invitation soon"]'),
    59 +    ("Five of Wands", 5, "Minor Arcana", "w05.jpg", '["Prepare for a fight with your best friend", "Remember: once you let words loose, you can\'t take them back"]'),
    60 +    ("Six of Wands", 6, "Minor Arcana", "w06.jpg", '["Someone is planning a party for you, but not everyone feels so good about your recent success", "Watch out for envious friends"]'),
    61 +    ("Seven of Wands", 7, "Minor Arcana", "w07.jpg", '["Don\'t be surprised by a personal attack", "Prepare to defend yourself or someone you love"]'),
    62 +    ("Eight of Wands", 8, "Minor Arcana", "w08.jpg", '["Watch for a surprising letter in the mail", "Your whole world is about to be turned on its ear"]'),
    63 +    ("Nine of Wands", 9, "Minor Arcana", "w09.jpg", '["Don\'t relax yet; there\'s more to come", "The test you\'re facing now is happening for one reason: to show you who your real friends are"]'),
    64 +    ("Ten of Wands", 10, "Minor Arcana", "w10.jpg", '["You\'re worn out", "Back off, take a time out, and let someone else handle things for a while"]'),
    65 +    ("Page of Wands", 11, "Minor Arcana", "w11.jpg", '["This card represents a young man or woman with a fiery, enthusiastic demeanor, likely born a Cancer, Leo, or Virgo, who wants to start a new relationship with you"]'),
    66 +    ("Knight of Wands", 12, "Minor Arcana", "w12.jpg", '["This card represents a man with a bold, passionate personality, likely born between July 12th and August 11th, who wants to sweep you off your feet"]'),
    67 +    ("Queen of Wands", 13, "Minor Arcana", "w13.jpg", '["This card represents a woman with an attractive, appealing personality, likely born between March 11th and April 20th, who wants to charm you into doing things her way"]'),
    68 +    ("King of Wands", 14, "Minor Arcana", "w14.jpg", '["This card represents an older man with a commanding, charismatic personality, likely born between November 13th and December 12th, who prefers to give directions and have them followed"]'),
    69 +    ("Ace of Pentacles", 1, "Minor Arcana", "p01.jpg", '["Your health will improve", "The check you\'re looking for really is in the mail"]'),
    70 +    ("Two of Pentacles", 2, "Minor Arcana", "p02.jpg", '["It\'s time to balance the budget", "Avoid the temptation to spend critical funds on frivolous goods"]'),
    71 +    ("Three of Pentacles", 3, "Minor Arcana", "p03.jpg", '["A high-dollar contract is in your future", "If you work hard, you\'ll succeed"]'),
    72 +    ("Four of Pentacles", 4, "Minor Arcana", "p04.jpg", '["A rainy day is coming—it\'s time to save"]'),
    73 +    ("Five of Pentacles", 5, "Minor Arcana", "p05.jpg", '["Finances are getting tighter", "Prepare for a setback"]'),
    74 +    ("Six of Pentacles", 6, "Minor Arcana", "p06.jpg", '["When you need help, ask for it", "Remember, though: what you receive may be limited by what you\'ve given to others in the past"]'),
    75 +    ("Seven of Pentacles", 7, "Minor Arcana", "p07.jpg", '["Things won\'t work out as expected", "Pick up the pieces and prepare to move on"]'),
    76 +    ("Eight of Pentacles", 8, "Minor Arcana", "p08.jpg", '["Stop over-analyzing, researching, and outlining", "Buckle down and get the work done"]'),
    77 +    ("Nine of Pentacles", 9, "Minor Arcana", "p09.jpg", '["Until you appreciate what you have, you won\'t have any luck getting more"]'),
    78 +    ("Ten of Pentacles", 10, "Minor Arcana", "p10.jpg", '["Big money is in the near future", "Expect a powerful blessing to come your way"]'),
    79 +    ("Page of Pentacles", 11, "Minor Arcana", "p11.jpg", '["This card represents a young man or woman with an earthy, practical demeanor, likely born an Aries, Taurus, or Gemini, who playfully encourages you to take financial or sexual risks"]'),
    80 +    ("Knight of Pentacles", 12, "Minor Arcana", "p12.jpg", '["A stingy person may chide you for spending money", "Be prepared to defend an economic or sexual decision"]'),
    81 +    ("Queen of Pentacles", 13, "Minor Arcana", "p13.jpg", '["This card represents a woman with an expansive, sensual nature, likely born between December 13th and 31st, who uses sensual appeal and the promise of reward to sway others to her point of view"]'),
    82 +    ("King of Pentacles", 14, "Minor Arcana", "p14.jpg", '["This card represents an older man with a financially, socially, and politically conservative spirit, likely born between August 12th and September 11th, who is known for putting his money where his mouth is"]'),
```

Reply:
doesn't apply for postgres